manifests/init.pp

```diff
@@ -136,324 +114,98 @@
         mode    => $pam_d_sshd_mode,
       }
 
-      case $::osfamily {
+      case $facts['os']['family'] {
```

Comment:
repeating the case statement inside itself seems wrong

Reply:
Well, that took more time for refactoring. Checkout the commit "Refactor case statements".